cli/gencert/gencert.go

```diff
@@ -112,6 +112,10 @@ func gencertMain(args []string, c cli.Config) (err error) {
 			return err
 		}
 
+		if len(req.Hosts) == 0 {
+			log.Warning("This certificate lacks a \"hosts\" field. This makes it unsuitable for websites.")
```

Comment:
Can you include the reference here to the CA/B Forum baseline requirements as a comment here? It was originally listed as a comment for CSRValidate.